templates/tutorialv2/stats/index.html

```diff
@@ -0,0 +1,234 @@
+{% extends "tutorialv2/base.html" %}
+{% load i18n %}
+{% load crispy_forms_tags %}
+{% load append_query_params %}
+{% load date_from_day_number %}
+{% load seconds_to_duration %}
+
+{% block title %}
+    {% trans "Statistiques du contenu" %}
+{% endblock %}
+
+{% block breadcrumb %}
+    <li><a href="{% url "content:stats-content" content.pk content.slug %}">{% trans "Statistiques" %}</a></li>
+    <li>{{ content.title }}</li>
+{% endblock %}
+
+{% block content_out %}
+    <!--TODO those should not be needed -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.20.1/moment.min.js"></script>
```

Comment:
Je n'avais pas réussi à me passer de ces imports. Normalement on devrait pouvoir fonctionner sans vue que les packages sont ajoutés dans le fichier json :thinking:

Reply:
@sandhose @abdelaz3r @Situphen une idée ? Quelle est la bonne pratique ici ?

Reply:
@Situphen j'ai tenté en ajoutant
```js
        require.resolve('moment'),
        require.resolve('chart.js'),
```

Mais sans succès, la console dev me parle d'une erreur "ReferenceError: require is not defined" pour Chart.js. (j'ai googlé, ca parle de node et de chart.bundle entre autre, j'ai tenté quelques trucs mais sans succès)

Reply:
Pas plus d'idée à tester ?